tests/e2e-inst-test.sh

```diff
@@ -129,7 +129,7 @@ for TEST in $TESTS; do
         --output option:parse-arguments \
         --log file:$SCRIPT_TMP_DIR/tracee-log-$$ \
         --signatures-dir "$SIG_DIR" \
-        --scope comm=echo,mv,ls,tracee,proctreetester,ping,ds_writer,fsnotify_tester \
+        --scope comm=echo,mv,ls,tracee,proctreetester,ping,ds_writer,fsnotify_tester,process_execute,tracee-ebpf \
```

Comment:
Unrelated to this PR, but I think that in other PR we should make the scope option trim the specified values to the 16 char limit, so passing an expected value like "process_execute_failed" would work since the trim will avoid the NUL overwrite.